README.md

```diff
@@ -12,41 +12,36 @@ Clone the repository
 
 Checkout the [Wiki](https://github.com/nimblehq/flutter-templates/wiki) page to access the full documentation.
 
-## Use the template
+## Template Usage
 
-### Setup a new project
+Before using the template, make sure you have the following prerequisites installed on your system:
 
-- To set up a new project from the template, run the command:
+- [Mason CLI](https://pub.dev/packages/mason_cli) 0.1.0-dev.44
+- Flutter 3.0.5
+- Flutter version manager (recommend): [fvm](https://fvm.app/)
 
-  - `$ make run PACKAGE_NAME={com.your.package} PROJECT_NAME={your_project_name} APP_NAME="{Your App Name}"`
+To set up a new project from the template, follow these steps:
 
-  - Then clean the project: `$ fvm flutter clean`
+- Open your terminal and run the following command:
 
-  - Re-fetch the project: `$ fvm flutter pub get`
+  - `$ mason make template`
```

Comment:
I think we should mention the Mason doc as a reference for the `make` command usage

```
- Open your terminal and run the following command:

  - `$ mason make template`

> Checkout the Mason documation for `make` command options & usage in detail [here](https://github.com/felangel/mason/tree/master/packages/mason_cli#overview)
```